src/js/control-bar/control-bar.js

```diff
@@ -67,6 +68,7 @@ ControlBar.prototype.options_ = {
     'descriptionsButton',
     'subtitlesButton',
     'captionsButton',
+    'tracksButton',
```

Comment:
Left in for testing to make sure they don't break each other, but I don't see a reason for them to be used like that for real.